app/src/main/java/com/example/tasky/agenda_details/presentation/AgendaDetailsRoot.kt

```diff
@@ -97,11 +97,7 @@ fun AgendaDetailsRoot(
             agendaDetailsViewModel.onDateSelected(selectedDate, dialogState, timeType)
         }
 
-    val titleText = if (!title.isNullOrEmpty()) title else getDefaultTitle(agendaItemType)
-    val itemDescription =
-        if (!description.isNullOrEmpty()) description else getDefaultDescription(agendaItemType)
-
-    val maxPhotosSelection = maxOf(2, maxPhotoCount - viewState.uriImageList.size)
+    val maxPhotosSelection = maxOf(2, maxPhotoCount - viewState.photos.size)
```

Comment:
This one I don't understand. If the user already picked 10 photos, then `maxPhotoCount - viewState.photos.size` will be 0 and the outcome will be that you can pick two more photos.. 🤔